File: 268-missing-number/missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int sum=0;
        int n=nums.length;
        for(int i=0;i<n;i++){
            sum+=nums[i];
        }
    int whole=(n*(n+1))/2;
    return whole-sum;
        
    }
}

File: 268-missing-number/missing-number.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n=len(nums)
        Sum=sum(nums)
        whole_sum=(n*(n+1))//2
        return whole_sum-Sum
        